packages/esm-patient-medications-app/translations/en.json

```diff
@@ -19,6 +19,8 @@
   "drugAlreadyPrescribed": "Already prescribed",
   "drugName": "Drug name",
   "drugOrders": "Drug orders",
+  "drugOrderValidationFailed": "drugOrderValidationFailed",
+  "drugOrderValidationFailedDescription": "drugOrderValidationFailedDescription",
```

Comment:
Translation extraction should be fixed in https://github.com/openmrs/openmrs-esm-patient-chart/commit/9d9c111e7abd104fdc501421526700f45703a40c. So if you rebase and run the script again, it should pick up the correct default values.